.gitmodules

```diff
@@ -6,3 +6,6 @@
 	path = libraries/appbase
 	url = https://github.com/eosio/appbase
         ignore = dirty
+[submodule "programs/snapshot"]
+	path = programs/snapshot
+	url = git@github.com:EOSIO/genesis.git
```

Comment:
Using an SSH URL is nice for us devs, but it breaks for people who don't have SSH keys set up on github. Best to use HTTPS URLs instead.

Reply:
You got it